ova/assets/customWelcomeKibana.txt

```diff
@@ -0,0 +1,65 @@
+/*------------- WAZUH -------------*/
+.wz-login {
```

Comment:
This is a CSS file instead of a TXT right?

Reply:
Yes, the content is inserted into another css file
Fixed in: https://github.com/wazuh/wazuh-packages/pull/746/commits/c143daa8de1e88ee8543d02fd0af75d4e6a0f732 and https://github.com/wazuh/wazuh-packages/pull/746/commits/689939c59e5e652e62470349288fd8d44d5f7f44